### DISCUSSION TITLE
PHP8.3 No matching or stable tool versions found

### DISCUSSION THREAD

**Discussion by RolphH** [renovatebot/renovate#26187] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

renovate:slim docker

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I have not seen this working

### Describe the problem

We have been using renovate without problems running sidecars for different php version. However, with the latest php8.3 release, renovate cannot update our packages.

It seems that renovate doesn't know the latest 8.3 stable release for PHP, and tries 8.2.13, which is breaking our specified constraints. This has tested both with `renovate --binary-source install` and `renovate --binary-source docker`. In both cases, it falls back to `php 8.2.13` instead of the latest `php 8.3.0` release.

### Relevant debug logs

<details><summary>Logs</summary>

```
WARN: No matching or stable tool versions found - using an unstable version (repository=lcs/container-service-api, branch=renovate/patch-require-dev-symplify-easy-coding-standard-12.x)
10:22:32         "toolName": "php",
10:22:32         "constraint": "8.3.*",
10:22:32         "stableVersion": "8.2.13"
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment]
I think this is waiting on https://github.com/php-build/php-build/pull/759

Cc @viceice

**Reply by viceice** [reply]
yes

**Reply by RolphH** [reply]
Not sure if you want to continue using that repo, activity seems very low looking at the pending PRs and Issues there

**Reply by rarkins** [reply]
Any similar tool which could replace it?

**Reply by HonkingGoose** [reply]
By the way, the "blocker" PR is merged now:

- https://github.com/php-build/php-build/pull/759

**Reply by viceice** [reply]
yes, and binaries are build, so should work now
